main.go

```diff
@@ -66,12 +66,14 @@ func main() {
 		sub.Route("/quickstarts", routes.MakeQuickstartsRouter)
 		sub.Route("/progress", routes.MakeQuickstartsProgressRouter)
 		sub.Route("/helptopics", routes.MakeHelpTopicsRouter)
+		sub.Route("/favorites", routes.MakeFavoriteQuickstartsRouter)
```

Comment:
I would hide this behind an env variable for now until the full workflow is finished.